example-definitions/cli-target-rest.yaml

```diff
@@ -2,4 +2,8 @@ authMode: "none"
 # Set authmode to 'basic' and uncomment username & password if you need authentication
 # username: "your-username"
 # password: "your-password"
-restURL: "http://localhost:4318/v1/traces"
+restURL:
+  baseURL: "http://localhost:4318"
```

Comment:
Minor nit, that using `URL` twice looks odd, maybe `baseURL` can become `base` or `host` etc.?

Reply:
Just `base` would be a bit confusing and `host` would be misleading.